src/vs/workbench/contrib/terminal/browser/terminalTabsWidget.ts

```diff
@@ -376,3 +351,38 @@ interface ITerminalTabEntryTemplate {
 	};
 	elementDispoables?: DisposableStore;
 }
+
+
+class TerminalTabsAccessibilityProvider implements IListAccessibilityProvider<ITerminalInstance> {
+	constructor(@ITerminalService private readonly _terminalService: ITerminalService) { }
+
+	getWidgetAriaLabel(): string {
+		return localize('terminal.tabs', "Terminal tabs");
+	}
+
+	getAriaLabel(instance: ITerminalInstance): string {
+		let ariaLabel: string = '';
+		const tab = this._terminalService.getTabForInstance(instance);
+		if (tab && tab.terminalInstances?.length > 1) {
```

Comment:
tried doing 

```
if (tab?.terminalInstances?.length > 1)  
```
here, but TS kept complaining

Reply:
Surprised it let you do the `tab.terminalInstances?.length > 1` since it would end up being number | undefined > 1. Should act correctly regardless though